### DISCUSSION TITLE
error message :- failed to fetch

### DISCUSSION THREAD

**Discussion by megarou76** [processing/p5.js-web-editor#2937] [General]
To be honest, I don't know how to describe this !?, but, the sketch of my project loads the first time & runs perfectly, when I re run it again, it gives me an error message :- failed to fetch, & then it freezes, & it tells me to refresh the page, I don't know if the bug/issue is from me or from the web editor?!!, I'm not sure!, what do ya think is the problem exactly?

**Comment by lindapaiste** [comment]
Can you please post the the link to the specific sketch which is having the problem?

I wonder if there is some problem with the code of your sketch which causes the preview to crash.

**Comment by megarou76** [comment]
p5.js Web Editor | Thunder Legends (Zero - (Experimental)) (p5js.org)

Thats the link to the sketch that has the problem.    On Tuesday, January 23, 2024 at 10:42:49 AM MST, Linda Paiste ***@***.***> wrote:  
 
 


Can you please post the the link to the specific sketch which is having the problem?

I wonder if there is some problem with the code of your sketch which causes the preview to crash.

**Reply by lindapaiste** [reply]
Looks like you replied via email and it didn't include the actual link.  It's probably best to reply on GitHub directly.  I need the URL of the sketch, like https://editor.p5js.org/lindapaiste2/sketches/eu_0qMTGI

**Reply by megarou76** [reply]
https://editor.p5js.org/megarou/sketches/AR4Nne3yU

ok, I hope this helps

**Reply by lindapaiste** [reply]
Hmmm...I haven't gotten it to crash yet.  I am getting an error relating to one of the sounds.

![image](https://github.com/processing/p5.js-web-editor/assets/28965286/73f058a6-57b8-48bb-b00c-6a925d64cc97)

This is unrelated by you have sooooo many variables.  I would recommend using nested arrays (an array of arrays).  Like instead of `let spritesheeta21 = []` etc. you would have a single `let spritesheeta = []` and then use `spritesheeta[21]` everywhere instead of `spritesheeta21`.
